From 450ef5884d9e93e6f0a2df48aae82f9ed239b394 Mon Sep 17 00:00:00 2001
From: David Edmundson <kde@davidedmundson.co.uk>
Date: Sat, 24 Mar 2018 07:26:49 +0000
Subject: [PATCH] Revert "use plasma_install_bundled_package"

This reverts commit e37156b404bffd4dc6fd0d1b2dbb5dc1ae0ddfa4.

Concept is fine, but currently we have severe CMake issues as well
as a major containment loading bug.

When re-reverting we need to be sure to test with clean install prefixes.

CC: plasma-devel@kde.org
---
 applets/CMakeLists.txt                      | 12 ++++++------
 applets/appmenu/CMakeLists.txt              |  2 +-
 applets/batterymonitor/CMakeLists.txt       |  2 +-
 applets/calendar/CMakeLists.txt             |  2 +-
 applets/devicenotifier/CMakeLists.txt       |  2 +-
 applets/digital-clock/CMakeLists.txt        |  2 +-
 applets/icon/CMakeLists.txt                 |  2 +-
 applets/notifications/CMakeLists.txt        |  2 +-
 applets/systemtray/CMakeLists.txt           |  2 +-
 applets/systemtray/container/CMakeLists.txt |  2 +-
 10 files changed, 15 insertions(+), 15 deletions(-)

diff --git a/applets/CMakeLists.txt b/applets/CMakeLists.txt
index 34b3a3994..956105e50 100644
--- a/applets/CMakeLists.txt
+++ b/applets/CMakeLists.txt
@@ -1,10 +1,10 @@
-plasma_install_bundled_package(activitybar org.kde.plasma.activitybar)
+plasma_install_package(activitybar org.kde.plasma.activitybar)
 add_subdirectory(icon)
 
-plasma_install_bundled_package(analog-clock org.kde.plasma.analogclock)
-plasma_install_bundled_package(mediacontroller org.kde.plasma.mediacontroller)
-plasma_install_bundled_package(panelspacer org.kde.plasma.panelspacer)
-plasma_install_bundled_package(lock_logout org.kde.plasma.lock_logout)
+plasma_install_package(analog-clock org.kde.plasma.analogclock)
+plasma_install_package(mediacontroller org.kde.plasma.mediacontroller)
+plasma_install_package(panelspacer org.kde.plasma.panelspacer)
+plasma_install_package(lock_logout org.kde.plasma.lock_logout)
 
 add_subdirectory(appmenu)
 add_subdirectory(systemmonitor)
@@ -12,7 +12,7 @@ add_subdirectory(batterymonitor)
 add_subdirectory(calendar)
 add_subdirectory(devicenotifier)
 add_subdirectory(digital-clock)
-plasma_install_bundled_package(clipboard org.kde.plasma.clipboard)
+plasma_install_package(clipboard org.kde.plasma.clipboard)
 
 if(NOT WIN32)
 #    #notifications
diff --git a/applets/appmenu/CMakeLists.txt b/applets/appmenu/CMakeLists.txt
index c1786eb41..731013ba7 100644
--- a/applets/appmenu/CMakeLists.txt
+++ b/applets/appmenu/CMakeLists.txt
@@ -1,4 +1,4 @@
 add_subdirectory(lib)
 add_subdirectory(plugin)
 
-plasma_install_bundled_package(package org.kde.plasma.appmenu)
+plasma_install_package(package org.kde.plasma.appmenu)
diff --git a/applets/batterymonitor/CMakeLists.txt b/applets/batterymonitor/CMakeLists.txt
index 78ff53b98..fb7aea96f 100644
--- a/applets/batterymonitor/CMakeLists.txt
+++ b/applets/batterymonitor/CMakeLists.txt
@@ -1 +1 @@
-plasma_install_bundled_package(package org.kde.plasma.battery)
+plasma_install_package(package org.kde.plasma.battery)
diff --git a/applets/calendar/CMakeLists.txt b/applets/calendar/CMakeLists.txt
index 7c08c68b6..869aa7155 100644
--- a/applets/calendar/CMakeLists.txt
+++ b/applets/calendar/CMakeLists.txt
@@ -11,4 +11,4 @@ target_link_libraries(plasma_applet_calendar
 
 install(TARGETS plasma_applet_calendar DESTINATION ${KDE_INSTALL_PLUGINDIR}/plasma/applets)
 
-plasma_install_bundled_package(package org.kde.plasma.calendar)
+plasma_install_package(package org.kde.plasma.calendar)
diff --git a/applets/devicenotifier/CMakeLists.txt b/applets/devicenotifier/CMakeLists.txt
index abb8952ad..23d70e2ef 100644
--- a/applets/devicenotifier/CMakeLists.txt
+++ b/applets/devicenotifier/CMakeLists.txt
@@ -1,3 +1,3 @@
-plasma_install_bundled_package(package org.kde.plasma.devicenotifier)
+plasma_install_package(package org.kde.plasma.devicenotifier)
 
 install(FILES test-predicate-openinwindow.desktop DESTINATION ${KDE_INSTALL_DATADIR}/solid/actions )
diff --git a/applets/digital-clock/CMakeLists.txt b/applets/digital-clock/CMakeLists.txt
index 878497428..3114564a4 100644
--- a/applets/digital-clock/CMakeLists.txt
+++ b/applets/digital-clock/CMakeLists.txt
@@ -1,3 +1,3 @@
 add_subdirectory(plugin)
 
-plasma_install_bundled_package(package org.kde.plasma.digitalclock)
\ No newline at end of file
+plasma_install_package(package org.kde.plasma.digitalclock)
\ No newline at end of file
diff --git a/applets/icon/CMakeLists.txt b/applets/icon/CMakeLists.txt
index c4793dc62..e4ffda679 100644
--- a/applets/icon/CMakeLists.txt
+++ b/applets/icon/CMakeLists.txt
@@ -17,4 +17,4 @@ target_link_libraries(plasma_applet_icon
 
 install(TARGETS plasma_applet_icon DESTINATION ${KDE_INSTALL_PLUGINDIR}/plasma/applets)
 
-plasma_install_bundled_package(package org.kde.plasma.icon)
+plasma_install_package(package org.kde.plasma.icon)
diff --git a/applets/notifications/CMakeLists.txt b/applets/notifications/CMakeLists.txt
index 7c60237c1..adfd60134 100644
--- a/applets/notifications/CMakeLists.txt
+++ b/applets/notifications/CMakeLists.txt
@@ -1,4 +1,4 @@
 add_subdirectory(lib)
 add_subdirectory(plugin)
 
-plasma_install_bundled_package(package org.kde.plasma.notifications)
+plasma_install_package(package org.kde.plasma.notifications)
diff --git a/applets/systemtray/CMakeLists.txt b/applets/systemtray/CMakeLists.txt
index 48d6a6fc2..4b5540b7f 100644
--- a/applets/systemtray/CMakeLists.txt
+++ b/applets/systemtray/CMakeLists.txt
@@ -1,6 +1,6 @@
 add_definitions(-DTRANSLATION_DOMAIN=\"plasma_applet_org.kde.plasma.private.systemtray\")
 
-plasma_install_bundled_package(package org.kde.plasma.private.systemtray)
+plasma_install_package(package org.kde.plasma.private.systemtray)
 
 set(systemtray_SRCS
     systemtray.cpp
diff --git a/applets/systemtray/container/CMakeLists.txt b/applets/systemtray/container/CMakeLists.txt
index 8d30a7651..d775ef755 100644
--- a/applets/systemtray/container/CMakeLists.txt
+++ b/applets/systemtray/container/CMakeLists.txt
@@ -1,5 +1,5 @@
 
-plasma_install_bundled_package(package org.kde.plasma.systemtray)
+plasma_install_package(package org.kde.plasma.systemtray)
 
 set(systemtraycontainer_SRCS
     systemtraycontainer.cpp
